railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -80,7 +80,8 @@ def self.plugin_rails_init(options)
       Minitest.backtrace_filter = ::Rails.backtrace_cleaner if ::Rails.respond_to?(:backtrace_cleaner)
     end
 
-    self.reporter.reporters.clear # Replace progress reporter for colors.
+    # Replace progress reporter for colors.
+    self.reporter.reporters.delete_if { |reporter| reporter.is_a?(SummaryReporter) || reporter.is_a?(ProgressReporter) }
```

Comment:
Shouldn't we use `kind_of?`, so any custom reporters subclassing `Minitest::SummaryReporter` and progress reporter are left in there?

On the other hand, if people have custom reporters shouldn't we clear those too?